### PR TITLE
[feat] add clickhouse support as a data source

### PR DESCRIPTION
**Pull request by Slach**
add clickhouse support as a data source, fix https://github.com/cuebook/CueObserve/issues/53

image
image
image
image

Signed-off-by: Slach <bloodjazman@gmail.com>

### CODE REVIEW

File: api/seeddata/connectionParam.json [hunk 1]
```diff
@@ -472,6 +472,145 @@
...
+          }
+        }
+      },
+      {
+        "model": "anomaly.connectionparam",
+        "pk": 31,
+        "fields": {
+          "name": "timeout",
+          "label": "Timeout for connect and send/receive in seconds",
+          "isEncrypted": true,
```
Comment (vincue) [inline api/seeddata/connectionParam.json:602]: Is it a mistake or necessary?

**Comment by Slach** [comment]
according to https://github.com/cuebook/CueObserve/pull/120/checks?check_run_id=3393765429
```
22 passed, 31 warnings in 16.99s
 ---------------- Coverage.xml file found --------------------- 
```

so look like in command 
`./api/test_reporter after-build --debug -r $CC_TEST_REPORTER_ID`
just empty secret
secrets.CODE_CLIMATE_TEST_REPORTER_ID

File: api/seeddata/connectionParam.json (hunk 1 above)
Comment (vincue) [reply api/seeddata/connectionParam.json:602]: `isEncrypted: true`

**Comment by vincue** [comment]
> according to https://github.com/cuebook/CueObserve/pull/120/checks?check_run_id=3393765429
> 
> ```
> 22 passed, 31 warnings in 16.99s
>  ---------------- Coverage.xml file found --------------------- 
> ```
> 
> so look like in command
> `./api/test_reporter after-build --debug -r $CC_TEST_REPORTER_ID`
> just empty secret
> `secrets.CODE_CLIMATE_TEST_REPORTER_ID`
> 
> I logged in codeclimate.com and defined API Key and setup it on my repo in secrets
> could we run PyTest github action manually again?

Not an issue, will look into it's alternative. Test cases will run again on `main branch`, can be merged for now.

File: api/seeddata/connectionParam.json (hunk 1 above)
Comment (Slach) [reply api/seeddata/connectionParam.json]: @vincue good catch, thanks a lot, fixed on https://github.com/cuebook/CueObserve/pull/120/commits/4c10cd804d6ca31a21d14d65670fa4c6b9a5d011

Comment (vincue) [reply api/seeddata/connectionParam.json]: Nice job there, thanks for contributing 🙏

**Pull request merged**